maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/dto/HRDetailDTO.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.justice.laa.maat.orchestration.dto;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+public class HRDetailDTO extends GenericDTO {
+
+    private Long id;
+    private FrequenciesDTO frequency;
+    private Date dateReceived;
+    private HRDetailDescriptionDTO detailDescription;
+    private String otherDescription;
+    private Currency amountNumber;
+    private Date dateDue;
+    private boolean accepted;
+    private HRReasonDTO reason;
+    private Timestamp timeStamp;
+    private String hrReasonNote;
+
+    public HRDetailDTO() {
+        reset();
+    }
+
+    public void reset() {

```

Comment:
Is there a specific reason why this only resets some of the fields and not all of them?